src/world/Action/EffectBuilder.cpp

```diff
@@ -176,6 +176,7 @@ std::shared_ptr< FFXIVPacketBase > EffectBuilder::buildNextEffectPacket( uint32_
     pHeader->effectCount = static_cast< uint8_t >( remainingTargetCount > packetSize ? packetSize : remainingTargetCount );
     pHeader->sourceSequence = m_sequence;
     pHeader->globalSequence = globalSequence;
+    pHeader->animationLockTime = 0.6f;
```

Comment:
This (and the other 2) are also wrong for the same reason. Regular attacks are 0.6s for the most part but a lot of others have different timings as well. Limit breaks are roughly 7.5s or so.

Believe they do this with an action category of sorts given how many shared values there are but I never really cared to confirm it

Reply:
Also ninja attacks (or just mudras, been a while) are handled differently for these too.

Reply:
didn't get anything other than 0.6 when I was capturing. well lb will be one I guess.

Reply:
Actually I got two non-0.6 results, see latest update.